exchanges/kraken/kraken_wrapper.go

```diff
@@ -634,6 +642,16 @@ func (k *Kraken) CancelAllOrders(_ *order.Cancel) (order.CancelAllResponse, erro
 		Status: make(map[string]string),
 	}
 
+	if k.Websocket.CanUseAuthenticatedWebsocketForWrapper() {
+		resp, err := k.wsCancelAllOrders()
+		if err != nil {
+			return cancelAllOrdersResponse, err
+		}
+
+		cancelAllOrdersResponse.Status["Count"] = fmt.Sprint(resp.Count)
```

Comment:
So the purpose of the status map is supposed to be that you get orderIDs and their status eg `["133713371337"]"Cancelled"`
I think it would be better to update `CancelAllResponse` struct with a `Count` int64 field and set that instead.